app/Http/Controllers/UserApiController.php

```diff
@@ -34,7 +34,12 @@ public function getAddress()
 
     public function getCommittees()
     {
-        return CommitteeMembership::where('user_id', Auth::id())->with('committee')->get();
+        return CommitteeMembership::where('user_id', Auth::id())->with('committee')->get()->where('is_society', false);
```

Comment:
I don't quite understand what you mean (can you nest queries in with statements? if so, cool, please show me :)). Since the `where` is behind the `get`, the 'is_society' query is done on the collection, so no queries should be performed.